### PR TITLE
Prep notes

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
* adds more detail to the demo prep notes

#13

### CODE REVIEW

**Comment by nelsonic** [comment]
@jackcarlisle looks good. the only thing I would _suggest_ is adding a few more sub-headings to make the content easier to scan.

File: README.md [hunk 1]
```diff
@@ -14,23 +14,28 @@ In our experience to date, this is delicate point for teams where it is relative
...
+    + If the demo isn't going to look like what the client was expecting, ensure that you explain this before you start so that they have a bit of time to digest why this is the case (***reduce the shock factor***)
 + Have all changes merged into `master` at least half a day before the demo
   + The rushed changes you make in the last half a day are usually the changes that break everything and cause stress during the demo
   + Your last day/half a day should be for polish and bug fixing from the merges, not for functionality
 + Breathe - this can be nerve wracking and taking some deep breaths will help slow you down!
++ Before the demo, set the expectation of how long you think the demo will be. This includes the demo itself + any questions you think the clients (or anyone else) might have. This shows that you’re organised and it prevents any inadvertent time wasting by time boxing it. It gives you a reference point that you can point to if you notice that you’re running over time. (_a passionate client can talk for hours if you let them!_)
```
Comment (iteles) [inline README.md]: The key action here is to send a calendar invite - that's the 'nicest' way to set the expectations.

**Comment by iteles** [comment]
Looks good! Agree with @nelsonic On the above 😊 

Thanks for this Jack!

**Comment by jruts** [comment]
I love you Jack <3

**Comment by iteles** [comment]
🎈 🎉 MERGED 🎉 🎈

**Pull request merged**